src/components/atoms/title/title.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { render, screen } from '@testing-library/react';
+import { Title } from './title';
+
+describe('Title', () => {
+  it('should render the title with a string passed into the title prop', () => {
+    const title = 'This is a title';
+    render(<Title title={title}/>);
+
+    expect(screen.getByText(title)).toBeInTheDocument();
+  });
+
+  it('should render the title with a array of strings passed into the title prop', () => { // whitespace issues
+    const titles = ['This', 'is', 'a', 'title'];
+    render(<Title title={titles}/>);
+
+    expect(screen.getByText('Thisisatitle')).toBeInTheDocument();
+  });
+
+  it('should render the title with a single content part passed into the title prop', () => {
+    const title = { type: 'Strong', content: 'This is a title' };
+    render(<Title title={title}/>);
+
+    expect(screen.getByText('This is a title')).toBeInTheDocument();
+  });
+
+  it('should render the title with an array of content parts passed into the title prop', () => { // whitespace issues
```

Comment:
This is exactly how encoda's output works at the moment. Which sort of makes sense - the array should only be used to decorate parts of a piece of content (e.g. Emphasis inside a paragraph). encoda seems to include a space in the string before and after, and this produces correct output. 

Given we're talking about HTML output though - whitespace is relatively insignificant, and it probably doesn't matter if you want to add extra space between array elements?